src/Features/Core/Portable/DocumentationComments/AbstractDocumentationCommentFormattingService.cs

```diff
@@ -289,6 +345,33 @@ public string Format(string rawXmlText, Compilation compilation = null)
             return state.GetText();
         }
 
+        public IEnumerable<TaggedText> Format(ISymbol symbol, SemanticModel semanticModel, int position, CancellationToken cancellationToken, SymbolDisplayFormat format = null)
+        {
+            var docSymbol = symbol;
+            if (symbol.GetTypeParameters().Any())
+            {
+                docSymbol = symbol.OriginalDefinition;
+            }
+
+            var rawXmlText = GetDocumentation(docSymbol, semanticModel.Compilation, cancellationToken);
```

Comment:
remove docSymbol entirely and just pass in `symbol.OriginalDefinition` here.

Reply:
This comment still applies. This is just keeping the original logic that we used to have in the caller and moving it here.